marlowe-dashboard-client/src/MainFrame/Types.purs

```diff
@@ -89,24 +90,46 @@ data Msg
 ------------------------------------------------------------
 data Action
   = Init
-  | ToggleOverlay Overlay
+  -- outside actions
```

Comment:
looking at this... outside actions is like WebSocket actions? or things that communicate with the server?. If they are WS, wouldn't it belong to the output message? In any case, I find strange to divide MainFrame actions to be inside/outside